From: Michael Biebl <biebl@debian.org>
Date: Sun, 29 Mar 2015 22:57:50 +0200
Subject: Don't make NetworkManager D-Bus activatable

If the NetworkManager daemon has been stopped manually we don't want it
to be autostarted by a client request.
---
 Makefile.am                    | 5 -----
 data/NetworkManager.service.in | 1 -
 2 files changed, 6 deletions(-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -3689,11 +3689,6 @@
 data/NetworkManager-dispatcher.service: $(srcdir)/data/NetworkManager-dispatcher.service.in
 	$(AM_V_GEN) $(data_edit) $< >$@
 
-data/org.freedesktop.NetworkManager.service: $(srcdir)/data/org.freedesktop.NetworkManager.service.in
-	$(AM_V_GEN) $(data_edit) $< >$@
-
-service_DATA += data/org.freedesktop.NetworkManager.service
-
 endif
 
 examples_DATA += data/server.conf
--- a/data/NetworkManager.service.in
+++ b/data/NetworkManager.service.in
@@ -24,6 +24,5 @@
 
 [Install]
 WantedBy=multi-user.target
-Alias=dbus-org.freedesktop.NetworkManager.service
 Also=NetworkManager-dispatcher.service
 
